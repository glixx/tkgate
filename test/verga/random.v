module top;

  initial
    begin
	$random(100);
	repeat (20)
	  $display($random);
    end

endmodule
