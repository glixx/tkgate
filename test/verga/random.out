44994396
745292378
254439306
10387488
1043940580
803319250
1235240792
1971986270
910734285
968931565
1619240249
337416194
1816805055
2138790185
1228100076
188749511
907455839
734581784
539825845
606952975
